aten/src/ATen/nnapi/nnapi_bind.cpp

```diff
@@ -214,6 +223,8 @@ static auto register_NnapiCompilation = [](){
     throw;
   }
 }();
+#else
+  #undef IS_IOS_NNAPI_BIND
```

Comment:
These two lines are not necessary. If it's not defined, you don't have to undefine it.

Reply:
I think they are necessary, because if it's not-not defined, then undefine it.

Reply:
Ah, make sense! I read it wrong.